03-your-first-test-assertion/src/test/java/tests/TestLogin.java

```diff
@@ -1,20 +1,26 @@
 package tests;
 
-import org.junit.Test;
-import org.junit.Before;
 import org.junit.After;
-import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import static org.junit.Assert.assertTrue;
 
 public class TestLogin {
 
     private WebDriver driver;
 
     @Before
     public void setUp() {
-        driver = new FirefoxDriver();
+        // Windows
+        //System.setProperty("webdriver.chrome.driver", "C:\\tools\\selenium\\chromedriver.exe");
+        // Mac
+        //System.setProperty("webdriver.chrome.driver", "/usr/local/bin/chromedriver");
+
+        driver = new ChromeDriver();
```

Comment:
(re: my comment in 02/, use this as the setup code in 02/ and move the geckodriver stuff to later examples.)